src/expr.cc

```diff
@@ -122,6 +135,10 @@ class SymRef : public Value {
   explicit SymRef(const Loc& loc, Symbol n) : Value(loc), name_(n) {}
   virtual ~SymRef() {}
 
+  virtual bool IsFunc(Evaluator*) const override {
+    return IsNumeric(name_.str());
```

Comment:
Can you add a comment about why this is true?

Reply:
done